torch/quantization/fx/prepare.py

```diff
@@ -458,7 +461,19 @@ def maybe_insert_output_observer_for_node(
                     get_default_output_activation_post_process_map().get(
                         matched_pattern,
                         act_post_process_ctr)
+
             observer = act_post_process_ctr()
+            if isinstance(observer, _InputEqualizationObserver):
+                # Find the next node in the graph
+                next_node = get_next_node(model, node)
```

Comment:
is this trying to find the observer for the node? I think we should use: https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/quantization_patterns.py#L79

Reply:
I wanted to get the observer of the next node. I wasn't sure if the maybe_get_last_node_only() function would get the next node's observer.